.gitlab-ci.yml

```diff
@@ -165,6 +203,16 @@ Integration:
         INTERNAL_NETWORK: ["true"]
         DISTRO_CODE: ["rhel_90"]
 
+Integration-nightly:
+  stage: test
+  extends: Integration
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "schedule" && $RUNNER == "aws/rhel-8.4-x86_64" && $SCRIPT != "libvirt.sh"'
+    # Run libvirt test on openstack because it has kvm support
+    - if: '$CI_PIPELINE_SOURCE == "schedule" && $RUNNER == "aws/rhel-8.4-x86_64" && $SCRIPT == "libvirt.sh"'
+      variables:
+        RUNNER: "openstack/rhel-8.4-x86_64"
```

Comment:
Looking at #1459 this seems like it needs to be an unconditional change in the Integration stage. I find it very weird if you are seeing bugs when running the nightly pipeline but not when running the regular one.

Reply:
Thanks for pointing this out. I've put some more time into investing why is this happening and it seems there's a bug in qemu-kvm because this issue appears after a newer version is used. I created rhbz#1975242 for it. I'm not sure if we should just switch to using Openstack until then or wait for some resolution :/ ... 

I guess I could also try to introduce a similar change in the regular pipeline so that only the libvirt test is ran on Openstack and do that for Centos stream as well as currently all of the Centos stream tests a re running in Openstack because of this bug.

Reply:
Ahh, looks like one of the rarest times we were saved because the NVRs we use were too old. What is called the `rhel-8` runner is still in fact 8.3. Will probably have to switch at the moment when we update the runners. Not a concern for now.